.github/workflows/maven.yaml

```
@@ -0,0 +1,60 @@
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      matrix:
+        java: [ 8, 11, 17, 21 ]
+        experimental: [false]
+        include:
+            - java: 24
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)